lib/mayaUsd/undo/OpUndoItems.h

```diff
@@ -427,6 +427,26 @@ class UfeSelectionUndoItem : public OpUndoItem
     MAYAUSD_CORE_PUBLIC
     static void select(const std::string& name, const Ufe::Selection& selection);
 
+    /// \brief create and execute a select node undo item and keep track of it.
+    /// The global selection is replaced.
+    MAYAUSD_CORE_PUBLIC
+    static void select(const std::string& name, const MDagPath& dagPath, OpUndoItemList& undoInfo);
+
+    /// \brief create and execute a select node undo item and keep track of it on the global list.
+    /// The global selection is replaced.
+    MAYAUSD_CORE_PUBLIC
+    static void select(const std::string& name, const MDagPath& dagPath);
+
+    /// \brief Create and execute a select node undo item and keep track of it.  The global
```

Comment:
Technically, this one and the next are not selecting node but clearing the selection, so the description should say "a clear  selection node undo item" instead of "a select node undo item".